app/views/admin/dashboard/index.html.erb

```diff
@@ -1,16 +1,81 @@
-<div class="top-row-links">
-  <hr>
-  <ul class="links-left">
-    <%= link_to "Turing Totals", goals_path %> &nbsp;
-    <%= link_to "Back to my goals", user_goals_path %>
-  </ul>
-  <hr>
-</div>
+<hr>
+  <div>
+    <%= link_to "Past goals" %> &nbsp;
+    <%= link_to "Turing Totals", goals_path %> &nbsp;&nbsp;
+  <span>
+    <%= image_tag('schedule-reminder.svg', size: '18x18') %>
+    <%= link_to "Sign up for text reminders", edit_user_path(current_user) %>&nbsp;&nbsp;
+  </span>
+  </div>
+<hr>
+  <h2>Welcome, <%= current_user.name %></h2>
+  <h3>This week is: <%=current_week.week_number.name%></h3><h4><%= current_week.start_date.strftime("%B %e") %>-<%= current_week.end_date.strftime("%e %Y") %></h4>
```

Comment:
this is a lot for your view to control. i suggest using a helper method to format these dates. OR you could create a method on "Week" that formats each date for you. either are fine options, but your view shouldn't be responsible for the formatting